starwars/src/components/Character.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './StarWars.css';
+
+
+class Character extends React.Component {
+
+    render() {
+        console.log(this.props.char.name);
+    return (
+            <ul>
+                <li><strong>Name: </strong> {this.props.char.name}</li>
```

Comment:
I notice that the phrase `this.props.char` is repeated a lot in this component. You can look into object destructuring to shorten your code a bit. Here's a link for that:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring
In your case, before your return statement you could do something like this:
`const { name, birth_year, gender, skin_color } = this.props.char;`
At that point, you could remove every other instance of the string "this.props.char" in your code